src/views/Lottery/components/PreviousRoundCard/Body.tsx

```diff
@@ -1,55 +1,131 @@
 import React from 'react'
-import { CardBody, Heading, Flex, Skeleton, Text, Box, Button, useModal } from '@pancakeswap/uikit'
+import styled from 'styled-components'
+import {
+  CardBody,
+  Heading,
+  Flex,
+  Skeleton,
+  Text,
+  Box,
+  Button,
+  useModal,
+  CardRibbon,
+  useMatchBreakpoints,
+} from '@pancakeswap/uikit'
 import { LotteryRound } from 'state/types'
-import { useGetUserLotteriesGraphData } from 'state/lottery/hooks'
+import { useGetUserLotteriesGraphData, useLottery } from 'state/lottery/hooks'
+import { LotteryStatus } from 'config/constants/types'
 import { useTranslation } from 'contexts/Localization'
 import WinningNumbers from '../WinningNumbers'
 import ViewTicketsModal from '../ViewTicketsModal'
 
+const StyledCardBody = styled(CardBody)`
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: auto;
+
+  ${({ theme }) => theme.mediaQueries.md} {
+    grid-column-gap: 72px;
+    grid-row-gap: 36px;
+    grid-template-columns: auto 1fr;
+  }
+`
+
+const StyedCardRibbon = styled(CardRibbon)`
+  right: -20px;
+  top: -20px;
+
+  ${({ theme }) => theme.mediaQueries.xs} {
+    right: -10px;
+    top: -10px;
+  }
+`
+
 const PreviousRoundCardBody: React.FC<{ lotteryData: LotteryRound; lotteryId: string }> = ({
```

Comment:
This component is shared with `YourHistory`. Lots of style tweaks here to match the designs for desktop sizes